src/lcd-tools.cpp

```diff
@@ -1,20 +1,31 @@
-#include "lcd-tools.h"
+#include "catfish-tools.h"
+#include "koi-tools.h"
 
 #include <CLI/CLI.hpp>
 
-using namespace AsteroidOS::LCD_Tools;
+#include <QSettings>
+
+const char* CONFIG_FILE = "/etc/asteroid/machine.conf";
+
 
 int main( int argc, char** argv ) {
-	CLI::App app{"lcd-tools: Various controls for the TicWatch LCD"};
 
-	app.add_flag_callback("--sync-time",SyncTime,"Sync lcd time with linux time");
+	QSettings m_settings(CONFIG_FILE, QSettings::IniFormat);
+	const QString machineCodename = m_settings.value("Identity/MACHINE", "unknown").toString();
 
-	try{
-		app.parse(argc,argv);
-		if (argc <= 1)
-			throw CLI::CallForAllHelp();
-	} catch (const CLI::ParseError &e){
-		return app.exit(e);
+	CLI::App app{"lcd-tools: Various controls for watches with a secondary displays"};
+	if (machineCodename == "koi") {
+		using namespace AsteroidOS::LCD_Tools::Koi;
+		app.add_flag("--sync-time",SyncTime,"Sync lcd time with linux time");
+		app.add_flag("--white-background",[](int){SetDisplayColor(true);},"(Koi) set display background to white");
```

Comment:
I'm wondering what the added benefit is of `(Koi)` in these cases.
They might as well be removed right?

Reply:
I just wanted to highlight the watch-specific features to users. These features are implemented here for testing and as a stopgap (ideally the lcd colour switching should also be presented somewhere in UI) and I want to make it clear to developers which features will only show up on their watch.

Reply:
Using that logic, the LCD timesync is also unique to `koi` :wink:
The `timepiece` mode is also available in an eventual update for `catfish`.
And obviously these won't be available on any non-second display watches.

Reply:
this is LCD-tools, it's only available on watches with additional timekeeping features that need time to be synchronised, so I think it's a reasonable assumption that all devices running the tool will need the `--sync-time` function. Catfish still doesn't have timepiece mode as of this PR, so the `(koi)` label should be removed when that's implemented.

Reply:
Another issue is that narwhal (which I intend to move here from hands-timesync for consistency) doesn't have a dedicated timepiece mode, it will just keep ticking after the system is shut down, assuming the hands haven't been locked into a certain location. If narwhal is added, the `timepiece` feature will need watch-specific labeling again. This gets even more complicated if the UI _does_ move the hands around - but we'll burn that bridge when we get to it.

Reply:
Sounds good :+1: